GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Command.cfg

```diff
@@ -530,7 +530,10 @@
 
 	@MODULE[ModulePartVariants]
 	{
```

Comment:
I don't think you meant to put this here

Reply:
This is still being changed

Reply:
Pls work